tests/redux-toolbelt-saga/makeAsyncSaga.js

```diff
@@ -0,0 +1,64 @@
+import 'babel-polyfill'
```

Comment:
is there a better place to put it?

Reply:
we generally put it in webpack as entry point. why do you need it for test? sound weird! tests can run w/o browser, so why a polyfill?

Reply:
await didnt work without it